rand_core/src/impls.rs

```diff
@@ -118,14 +118,14 @@ pub fn fill_via_u64_chunks(src: &[u64], dest: &mut [u8]) -> (usize, usize) {
 pub fn next_u32_via_fill<R: RngCore + ?Sized>(rng: &mut R) -> u32 {
     let mut buf = [0; 4];
     rng.fill_bytes(&mut buf);
-    u32::from_le_bytes(buf)
+    u32::from_ne_bytes(buf)
 }
```

Comment:
?

Indeed, the old version didn't byte-swap. But `fill_bytes_via_next` does byte-swap. Weird?

Reply:
I'm also puzzled.